cerberus-log-processor-lambda/application.yaml

```diff
@@ -36,18 +39,38 @@ Parameters:
   SlackWebHookUrl:
     Type: String
     Description: The Slack HTTP endpoint to which the Lambda will post rate limit notifications
+  AthenaDatabaseName:
+    Type: String
+    Description: The name of ALB log's Athena database
+  AthenaTableName:
+    Type: String
+    Description: The name of ALB log's Athena table
+  AthenaQueryResultBucketName:
+    Type: String
+    Description: The name of Athena query result's S3 bucket
+  AlbLogBucket:
+    Type: String
+    Description: The name of ALB log's S3 bucket
+  AlbLogBucketArn:
+    Type: String
+    Description: The ARN of ALB log's S3 bucket
 Resources:
   LambdaLogProcessorFunction:
     Type: AWS::Serverless::Function
     Properties:
       Description: Function for auto black listing ips that are misbehaving
       CodeUri: @@CODE_URI@@
       Handler: >-
-         com.nike.cerberus.lambda.waf.handler.ALBAccessLogEventHandler::handleNewS3Event
+         com.nike.cerberus.lambda.waf.handler.ALBAccessLogEventHandler::handleScheduledEvent
       Runtime: java8
       MemorySize: 512
       Timeout: 60
       Role: !GetAtt LogProcessorLambdaRole.Arn
+      Events:
+        Timer:
+          Type: Schedule
+          Properties:
+            Schedule: rate(5 minutes)
```

Comment:
Maybe not now, but it would be nice to make this configurable later

Reply:
We can make it configurable later for sure. I'm using 5 minutes since the ALB log files are generated at the same rate.